Rakefile

```
@@ -72,12 +72,15 @@ end
 
 desc 'Confirm documentation is up to date'
 task confirm_documentation: :generate_cops_documentation do
-  _, _, _, process =
-    Open3.popen3('git diff --exit-code docs/')
-
-  unless process.value.success?
-    raise 'Please run `rake generate_cops_documentation` ' \
-          'and add docs/ to the commit.'
+  stdout, _stderr, status =
+    Open3.capture3('git diff --exit-code docs/')
+
+  unless status.success?
+    puts 'Documentation is out of sync:'
```

```suggestion
    warn 'Documentation is out of sync:'
```

Same as `warn` for us running on CI ? (Skipping the discussion on stdout vs stderr)


This line and the three below

---

Fixed it 👍🏻  Thank you.